drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -352,15 +365,16 @@ def student_view(self, context):
             'public/js/drag_and_drop.js',
         ]
 
-        statici18n_js_url = self._get_statici18n_js_url()
-        if statici18n_js_url:
-            js_urls.append(statici18n_js_url)
-
         for css_url in css_urls:
             fragment.add_css_url(self.runtime.local_resource_url(self, css_url))
         for js_url in js_urls:
             fragment.add_javascript_url(self.runtime.local_resource_url(self, js_url))
 
+        if static_i18n_js_url := self.get_javascript_i18n_catalog_url():
+            fragment.add_javascript_url(static_i18n_js_url)
+        elif deprecated_i18n_js_url := DragAndDropBlock.get_deprecated_i18n_js_url():
+            fragment.add_javascript_url(self.runtime.local_resource_url(self, deprecated_i18n_js_url))
```

Comment:
@OmarIthawi, nit: instead of duplicating this code, we could add it to the `get_javascript_i18n_catalog_url` method (before `return None`). The removal of this deprecated feature will be easier this way.

Reply:
done. I'm rebasing to fix tests.